challenges/objects-arrays.js

```diff
@@ -80,8 +121,11 @@ zooAnimals = [{"animal_name":"Jackal, asiatic","population":5,"scientific_name":
 
 // The zoos need a list of all their animal's names converted to lower case.  Create a new array named lowerCase and map over each name to convert them all to lower case.  Log the resut.
 let lowerCase = [];
+
 console.log(lowerCase); 
 
 // The zoos need to know their total animal population across the United States.  Add up all the population numbers from all the zoos using the .reduce() method.
 let populationTotal = [];
-console.log(populationTotal);
\ No newline at end of file
+populationTotal.push(zooAnimals[i].population);
+const reducer=(accumalator, currentValue)=> accumalator + currentValue;
+console.log(populationTotal.reduce(reducer));
```

Comment:
Here it's the same concept as reduce will return an array, so you do not need the brackets.  Pass animal, and total as parameters on the reduce method.  You could call this anything as long as it makes sense.   Then use that to iterate each animal in the list by referencing population.  Reduce adds that to a single sum.

let populationTotal = zooAnimals.reduce(function(animal, total) {
  return animal + total.population;
}, 0);
console.log(populationTotal);